### ISSUE TITLE
Can't use a large cache size limit

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epidatr#188]
``` r
epidatr::set_cache(max_size = 10000L)
#> Warning in cachem::cache_disk(dir = cache_dir, max_size = as.integer(max_size *
#> : NAs introduced by coercion to integer range
```

<sup>Created on 2023-09-28 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

I believe R always uses 32-bit integers, so `10000L * 1024^2` is too large.

**Comment by dsweber2** [comment]
So... that's 10,000GB. I'm not sure we have 10TB worth of data, and caching that is... not worth it. XD

Was it not clear that `max_size` is expressed in GB?

**Comment by dsweber2** [comment]
Ok, so @dshemetov pointed out that we are in fact expressing in MB, so this is definitely valid, as it's only 10GB, which would be reasonable to support.

However, this is less our problem and more `cachem`'s, as they're the ones expressing things in 32-bit integers.

**Comment by brookslogan** [comment]
No, it's in MB?
```
max_size: the size of the entire cache, in MB, at which to start
          pruning entries. By default this is '1024', or 1GB. The
          environmental variable is 'EPIDATR_CACHE_MAX_SIZE_MB'.
```

**Comment by dsweber2** [comment]
You're right, not sure what I was thinking at the time. The PR Dmitry made should fix the error you were running into though.

**Comment by brookslogan** [comment]
Looks like it.  This is indeed closed in #189.  Somehow read over this and thought it was closed as not planned.
